package.json

```diff
@@ -24,5 +24,9 @@
   "devDependencies": {
     "aws-lambda-mock-context": "^1.1.0",
     "tape": "^4.4.0"
+  },
+  "dependencies": {
+    "babel-core": "^6.4.5",
```

Comment:
They don't appear to be using Babel/ES6 ... do you have access to the "Canary" project? (see gitter chat)